cypress/integration/features/deals/choose-deal-type.feature

```diff
@@ -1,9 +1,21 @@
 Feature: Choose Deal type
-    Background:
-      Given I navigate to the Deals home page
-      And I navigate to the All Deals page
+  Background:
+    Given I navigate to the Deals home page
+    And I navigate to the initiate a deal page by clicking Initiate a Deal
+    
+  Scenario: I can see correct deal types
+    Then I can see Token Swap deal type
+    And I can see Joint Venture deal type
 
-    Scenario: Read about deal types
-      Then I can read about the deal types
+  Scenario: I can select Token Swap
+    When I click select on Token Swap card
+    Then I am redirected to Do you have a partner page
 
-    Scenario: Choose deal types
\ No newline at end of file
+  Scenario: I can see correct Token Swap types
+    Given I click select on Token Swap card
+    Then I can see Open Proposal and Partnered Deal
+    
+  Scenario: I can see select Open Proposal
+    Given I click select on Token Swap card
+    And I click select Open Proposal
+    Then I am redirected to Open Proposal wizard
```

Comment:
```suggestion
    Then I can view the Open Proposal wizard
```
**Reason:** An assumption is made about how the UI is done (could open in a modal as well?)

Note: I'm not 100% happy with my suggestion either

Reply:
Makes sense